funcbench/manifests/benchmark/2_job.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: workflow
+  namespace: funcbench-{{ .PR_NUMBER }}
+data:
+  event.json: |
+    {
+      "action": "created",
+      "issue": {
+        "number": {{ .PR_NUMBER }}
+      },
+      "repository": {
+        "name": "{{ .GITHUB_REPO }}",
+        "owner": {
+          "login": "{{ .GITHUB_ORG }}"
+        }
+      }
+    }
+  BRANCH: {{ .BRANCH }}
+  RACE: {{ .RACE }}
+  REGEX: {{ .REGEX }}
```

Comment:
what are these for?

Reply:
They are the inputs of `funcbench`.
See https://github.com/prometheus/prombench/blob/aad5c780d5cb2ef3e866f49f2b4a500416217bf1/funcbench/main.go#L231-L235
and 
https://github.com/prometheus/prombench/blob/aad5c780d5cb2ef3e866f49f2b4a500416217bf1/funcbench/main.go#L165

Reply:
why not set them as env vars?

Reply:
@nevill commentMonitor no longer works by sharing variables in files. To avoid confusion, I think it'll be best to replace these with command line arguments or environment variable whichever is more preferable here. I might be misunderstanding aswell, can you share the github actions workflow file used to trigger funcbench here?

Reply:
yes actually I also meant command line variable

Reply:
@geekodour the workflow file is [here](https://github.com/nevill/prometheus/blob/master/.github/workflows/benchmark.yml)

Reply:
this is to be changed to cli args right?

Reply:
right, will update this soon